types/evernote/index.d.ts

```diff
@@ -100,12 +105,12 @@ export namespace NoteStore {
         });
     }
     class NoteCollectionCounts {
-        notebookCounts?: Map<Types.Guid, number>;
-        tagCounts?: Map<Types.Guid, number>;
+        notebookCounts?: Dictionary<Types.Guid, number>;
+        tagCounts?: Dictionary<Types.Guid, number>;
```

Comment:
```suggestion
        notebookCounts?: Record<Types.Guid, number>;
        tagCounts?: Record<Types.Guid, number>;
```
No need for custom type, the `Record` will work just fine here, properly exposing intent of the type:
https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeystype